Add length limitation for description member (#12033)
mdn/content: files/en-us/mozilla/add-ons/webextensions/manifest.json/description/index.md

  </tbody>
</table>

> A short description(no more than 132 characters) of the extension, intended for display in the browser's user interface.

Reviewer: ```suggestion
A short description (no more than 132 characters) of the extension, intended for display in the browser's user interface.
```

Author: Added the space and also removed the `standard-table` class as the styles for that are now directly applied on `<table>`

  </tbody>
</table>

> A short description (no more than 132 characters) of the extension, intended for display in the browser's user interface.
> 
> > Note that browsers might impose different limits for this field.

Reviewer 2: I would suggest rephrasing like this:
```suggestion
A short description of the extension, intended for display in the browser's user interface. In Firefox this value can be up to 132 characters. The limit in other browsers may differ.
```

Author: Added chrome also according to - https://developer.chrome.com/docs/extensions/mv3/manifest/description/
